services/web/Makefile

```diff
@@ -28,15 +29,33 @@ include ../../.make/generate.mk
 ci-go-generate: # CI runs ci-node-generate automatically before this target
 
 .PHONY: ci-node-generate
-ci-node-generate: pull-assets
+ci-node-generate: download-assets
```

Comment:
I'd very much like it if our default would be `pull-assets`. We can then use `download-assets` when crafting a release.